httpserver/src/test/kotlin/ServerTest.kt

```diff
@@ -27,7 +27,6 @@ open class ServerTest {
       val serverConfig = TbdexHttpServerConfig(
         port = 8080,
         pfiDid = TestData.pfiDid.uri,
-        balancesEnabled = true
```

Comment:
This change is not related to this PR, but is necessary to pass CI. I think when we merged #217 (and knowingly broke CI on main), there was one more broken thing than we realized.

Reply:
oops, that was my bad! i flip flopped but ended up removing that config. thanks @diehuxx